FontoXPathCSharp/FontoXPathCSharp.csproj

```diff
@@ -2,6 +2,8 @@
 
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
+        <RuntimeFrameworkVersion>6.0.22</RuntimeFrameworkVersion>
+		<Version>7.0.0</Version>
```

Comment:
Please remove the `<Version>7.0.0</Version>` it is not needed for the dotnet upgrade and we don't need to specify version yet in the stage we're at imo